.github/dependabot.yml

```diff
@@ -3,10 +3,10 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
I like when the actions is on a different schedule than the pip dependencies. It makes it less spammy on Monday.

If we set it to daily you have time to work through that during the week.